docs/codacy-api/using-the-codacy-api.md

```diff
@@ -16,7 +16,12 @@ Codacy supports two API versions but we strongly recommend using the new API v3
     <tr>
       <th>Endpoint documentation</th>
       <td><a target="_blank" href="https://api.codacy.com/api/api-docs">https://api.codacy.com/api/api-docs</a></td>
-      <td><a target="_blank" href="https://api.codacy.com/swagger">https://api.codacy.com/swagger</a></td>
+      <td><a target="_blank" href="https://api.codacy.com/api-docs">https://api.codacy.com/api-docs</a></td>
+    </tr>
+    <tr>
+      <th>OpenAPI definition</th>
+      <td><a target="_blank" href="https://api.codacy.com/api/api-docs/swagger.yaml">https://api.codacy.com/api/api-docs/swagger.yaml</a></td>
+      <td><a target="_blank" href="https://api.codacy.com/api-docs/swagger.yaml">https://api.codacy.com/api-docs/swagger.yaml</a></td>
```

Comment:
I am still a bit reluctant of putting this here.. 

Although our v2 api docs match exactly this definition, those are not in a very good state and dont match the actual endpoints in some aspects (mainly authentication and return values). This makes the docs still useful (they document existing endpoints although not to the correct extension) but implies that the swagger file contents may deceive the user at times since they may mismatch with the actual api.

IMO, we should work on this swagger file and bring it up to date to also have correct documentation for this API, and only after make available this in our public docs. 
The reality is that we have been deprecating this API, so not much care has been given to it in the latest years and we have been providing it as-is, and the docs are not great. 

If we want to fix it is not breaking for development, since this spec was maintained manually and dissociated to the actual implementation (using the Play Framework) - I can bootstrap this during my next offline day (this wed) if you are along for the ride @prcr 

any takes on this @codacy/bigbang ?

Reply:
I think that those are very valid concerns, @pedrocodacy. Essentially, relying on the OpenAPI definition for v2 of the API could be more confusing and detrimental than helpful in some cases.

I agree that we don't include that definition URL for now, or even indefinitely, depending on what's possible to improve. In the meantime, I commented out the URL on the page. :+1:

I'm planning to work on [another (big) task](https://codacy.atlassian.net/browse/DOCS-25) on my offline day, but do let me know if you choose to tackle some improvements here and if I can help asynchronously @pedrocodacy. :rocket: